src/providers/ldap/sdap_async.c

```diff
@@ -1805,7 +1805,8 @@ static void generic_ext_search_handler(struct tevent_req *subreq,
     talloc_zfree(subreq);
     if (ret != EOK) {
         DEBUG(SSSDBG_OP_FAILURE,
-              "sdap_get_generic_ext_recv failed [%d]: %s\n",
+              "sdap_get_generic_ext_recv failed [%d]: %s "
+              "[ldap_search_timeout]\n",
```

Comment:
`sdap_get_generic_ext_recv()` doesn't always fail due to a timeout, so I think that the debug message should differ depending on the error.